packages/common/src/graphiqlHTML.ts

```diff
@@ -0,0 +1 @@
+export default '<!DOCTYPE html><html lang=en><head><meta charset=utf-8><title>__TITLE__</title><link rel=icon href=https://www.graphql-yoga.com/favicon.ico><link rel=stylesheet href=https://unpkg.com/@graphql-yoga/graphiql@2.4.0/dist/style.css></head><body id=body class=no-focus-outline><noscript>You need to enable JavaScript to run this app.</noscript><div id=root></div><script type=module>import{renderYogaGraphiQL}from"https://unpkg.com/@graphql-yoga/graphiql@2.4.0";renderYogaGraphiQL(root,__OPTS__)</script></body></html>'
```

Comment:
Do we need to commit this? Aren’t we already doing  it somewhere?

Reply:
No it was commited by mistake :)